content/react/pt/get-started.md

```diff
@@ -6,20 +6,20 @@ description: "Configuração do React Storybook no ambiente de desenvolvimento"
 
 # Introdução
 
-Storybook funciona em paralelo á aplicação em modo de desenvolvimento.
-Ajuda na construção de componentes de interface de utilizador isolados de qualquer lógica e contexto da aplicação.
-Esta edição de Aprendizagem de Storybook é destinada para React.
-Encontram-se disponíveis outras edições quer para [Vue](/vue/pt/get-started), quer para [Angular](/angular/pt/get-started).
+Storybook funciona em paralelo à aplicação em modo de desenvolvimento.
+Ajuda na construção de componentes de interface de usuário, isolados de qualquer lógica e contexto da aplicação.
+Esta edição de Aprenda Storybook é destinada para React.
+Encontram-se disponíveis outras edições tanto para [Vue](/vue/pt/get-started), quanto para [Angular](/angular/pt/get-started).
```

Comment:
In this case using either phrasing is correct, there's no need to change it.

Reply:
Sorry, it is not common in Brazil to use/read "Utilizador".